index.html

```diff
@@ -1751,11 +1716,7 @@
         confidentiality and authenticity between corresponding <a>presentation
         sessions</a>.
       </p>
-      <p class="open-issue">
-        <a href="https://github.com/w3c/presentation-api/issues/80">ISSUE 80:
-        Define security requirements for messaging channel between secure
-        origins</a>
-      </p>
+      <div class="issue" data-number="80"></div>
```

Comment:
https://github.com/w3c/presentation-api/issues/80

Please note the spec is rendered as follows https://rawgit.com/anssiko/presentation-api/issue-tracker-integration/index.html with data pulled from GH issues using GitHub API.